docs/new-ansible-and-freezes.md

```diff
@@ -0,0 +1,146 @@
+
+    ``` sh
+    cat <<EOF >${MAJOR_VERSION}/changelog.yaml
+    ---
+    ancestor: ${PREVIOUS_MAJOR_VERSION}.0.0
+    releases: {}
+    EOF
+    ```
+
+6. Create a blank `validate-tags-ignores` file
+
+    ``` sh
+    touch "${MAJOR_VERSION}/validate-tags-ignores"
+    ```
+
+7. Add the next major version to ansible-build-data's CI
+
+    Open `.github/workflows/antsibull-build.yml` and the following block to the
+    matrix:
+
+    ``` yaml
+    - name: Ansible ${MAJOR_VERSION}
+      ansible_version: ${MAJOR_VERSION}.99.0
+      ansible_major_version: ${MAJOR_VERSION}
+    ```
+
+8. Commit the changes
+
+    ``` sh
+    git add "${MAJOR_VERSION}"
+    ```
+
+9. Submit a PR against ansible-build-data
+
+
+## Freeze release
+
+<!-- TODO: Write a script, playbook, or antsibull-build subcommand to automate this -->
+
+Beyond the regular [Ansible Release Process](release-process.md), X.Y.0
+releases require special handling before running the release playbook.
+
+1. Determine the previous and the current releases
+
+    ``` sh
+    MAJOR_VERSION=9
+    VERSION=9.0.0
+    PREVIOUS_VERSION=9.0.0rc1
+    ```
+
+2. Set up your Git clones and release venv as outlined in the [Ansible Release Process](release-process.md) document.
+
+3. Copy over the previous release's deps and galaxy files.
+
+    ``` sh
+    cp "ansible-${PREVIOUS_VERSION}.yaml" "ansible-${VERSION}.yaml"
+    cp "ansible-${PREVIOUS_VERSION}.deps" "ansible-${VERSION}.deps"
+    ```
+
+4. Edit the current ansible version in the deps file
+
+    ``` sh
+    sed -i "s|^_ansible_version:.*$|_ansible_version: ${VERSION}|" \
+       "ansible-${VERSION}.deps"
+    ```
+
+5. Add a changelog entry for the new release
+
+    Open `${MAJOR_VERSION}/changelog.yaml` and the following block to the
+    releases table:
+
+    ``` yaml
+     ${VERSION}:
+       changes:
+         release_summary: 'Release Date: ${RELEASE_DATE}
+           `Porting Guide <https://docs.ansible.com/ansible/devel/porting_guides.html>`_'
+       release_date: '${RELEASE_DATE}'
+    ```
+
+    The release date should be formatted as `YYYY-MM-DD`.
+
+6. Manually update specific collection versions if needed.
+
+    In some cases, it may be necessary to update certain collections if, for
+    example, a serious bug is found is one of the collections.
+    In that case, open up the deps and galaxy files copied over in the step 2
+    and change the versions for the collections in question.
+    Make sure that the versions in both files are consistent.
+
+7. Generate the tags data file
+
+    ``` sh
+    antsibull-build validate-tags \
+        --data-dir . \
+        --ignores-file validate-tags-ignores \
+        --output "ansible-${VERSION}-tags.yaml" \
+        "${VERSION}"
+    ```
+
+8. Run the the release playbook as outlined in [Ansible Release Process](release-process.md).
+   Make sure to use pass `-e antsibull_data_reset=false` to preserve the
+   ansible-build-data modifcations.
```

Comment:
The release playbook runs `antsibull-build prepare` anyway, which adds a changelog fragment (so step 5 isn't needed) and overwrites the `.deps` and `.yaml` files created in step 4 and 6, and also the tags data file from step 7.

Reply:
The playbook is idempotent, and it will not run those steps if the `.deps` file is present. See https://github.com/ansible-community/antsibull/blob/9cb292752349bdc5f1f5db0cadcf0f61e62e1de2/roles/build-release/tasks/build.yaml#L60.

Reply:
Oh right, I missed that... Sorry!